tests/test_scene_rendering/conftest.py

```diff
@@ -1,15 +1,15 @@
 import pytest
-import os
 
+from pathlib import Path
 
 from manim import file_writer_config
 
 
 @pytest.fixture
 def manim_cfg_file():
-    return os.path.join(os.path.dirname(__file__), "manim.cfg")
+    return str(Path(__file__).parent.joinpath("manim.cfg"))
```

Comment:
```suggestion
    return str(Path(__file__).parent / "manim.cfg")
```
What I really love about the pathlib module is the `/` operator, so my suggestion would be to use this instead of joinpath